src/shaders/_prelude.vertex.glsl

```diff
@@ -32,6 +32,19 @@ vec3 mix_globe_mercator(vec3 globe, vec3 mercator, float t) {
 #endif
 }
 
+#ifdef PROJECTION_GLOBE_VIEW
+vec3 extrudeOnGlobeSurface(vec2 extrude, float scale, vec3 pos_nomal, vec3 up_dir, float zoom_transition) {
+    // Apply extra scaling to extrusion to cover different pixel space ratios (which is dependant on the latitude)
+    extrude *= scale;
+
+    vec3 normal = normalize(mix(pos_nomal / 16384.0, up_dir, zoom_transition));
```

Comment:
scaling of `extrude` variable and division of `pos_normal` could be done by the caller before invoking this function